### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by nan-yu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,11 +159,11 @@ The controller doesn't do much at the moment. However, if you'd like to build an
...
 
 ```commandline
-make docker-build IMG=<image>
-make docker-push IMG=<image>
+make docker-build CONTROLLER_IMG=<image>
+make docker-push CONTROLLER_IMG=<image>
 ```
 
-## Installing the CRD
+## Installing the CRD together with the controller
```
Comment (barney-s) [inline README.md]: This is just the controller.

Comment (nan-yu) [reply README.md]: It includes both the controller and the CRD.
`$(TOOLBIN)/kustomize build config/default | kubectl apply -f -` applied on `config/default`, which also includes the CRD. See: https://github.com/kubernetes-sigs/application/blob/master/config/default/kustomization.yaml#L21

Here is an example of the command output, which verified the CRD is created.

$ make deploy
cd config/manager && /usr/local/google/home/nanyu/go/src/sigs.k8s.io/application/hack/tools/bin/kustomize edit set image controller=gcr.io/nanyu-gke-dev/application-controller-amd64:dev
/usr/local/google/home/nanyu/go/src/sigs.k8s.io/application/hack/tools/bin/kustomize build config/default | kubectl apply -f -
namespace/application-system created
**customresourcedefinition.apiextensions.k8s.io/applications.app.k8s.io created**
role.rbac.authorization.k8s.io/application-leader-election-role created
clusterrole.rbac.authorization.k8s.io/application-manager-role created
clusterrole.rbac.authorization.k8s.io/application-proxy-role created
rolebinding.rbac.authorization.k8s.io/application-leader-election-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/application-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/application-proxy-rolebinding created
service/application-controller-manager-metrics-service created
service/application-controller-manager-service created
deployment.apps/application-controller-manager created

**Pull request merged**